usr/src/uts/common/os/kmem.c

```diff
@@ -2185,24 +2186,46 @@ kmem_depot_ws_zero(kmem_cache_t *cp)
 }
 
 /*
+ * Setting it this high (1GB) casues the explicit kpreempt to have
+ * very little effect.  Setting it to 1MB causes us to call
+ * kpreempt() approximately every 2 milliseconds.
```

Comment:
Agreed that the second sentence is a little too specific to be true long term. How about this?
```
Setting it this high (1GB) causes the explicit kpreempt() to have very little effect.  To make calls to kpreempt() more frequent, use a smaller value. A few megabytes is usually small enough that it will run hundreds of times per second.
```